api/scripts/update-certification-challenges-with-skill-id.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const _ = require('lodash');
+const Promise = require('bluebird');
+const { knex } = require('../lib/infrastructure/bookshelf');
+const skillDatasource = require('../lib/infrastructure/datasources/airtable/skill-datasource');
+
+async function getDistinctAssociatedSkillNamesForNullSkillId() {
+  const result = await knex.raw('SELECT DISTINCT ?? FROM ?? WHERE ?? IS NULL', ['associatedSkillName', 'certification-challenges', 'associatedSkillId']);
+  return _.map(result.rows, 'associatedSkillName');
+}
+
+async function mapSkillNamesWithId(skillNames) {
+  return Promise.map(skillNames, async (skillName) => {
+    const skills = await skillDatasource.findByName(skillName);
+    if (_.isEmpty(skills)) {
+      return {
+        skillName,
+        skillId: null,
+      };
+    }
+    return {
+      skillName,
+      skillId: skills[0].id,
+    };
+  }, { concurrency: 100 });
+}
+
+async function updateAllAssociatedSkillId(skillsMap) {
+  try {
+    await knex.transaction(async (trx) => {
```

Comment:
Le mot clé `await` n'est pas présent dans cette fonction donc pas besoin d'utiliser `async`
```suggestion
    await knex.transaction((trx) => {
```

Il y a le mot clé aussi à la ligne suivante sans utilisation de await.

Reply:
:+1: